public/js/gogs.js

```diff
@@ -840,6 +840,26 @@ function initEditor() {
         else {
             codeMirrorEditor.setOption("lineWrapping", false);
         }
+
+				$.getJSON($editFilename.data('ec-url')+$editFilename.val(), function(editorconfig) {
```

Comment:
You was right, it was wrong, I messed up with my editor settings. :sweat_smile: 

It's fixed now.